browser/ui/android/omnibox/java/res/drawable/ic_brave_ai_color.xml

```diff
@@ -0,0 +1,31 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We usually add headers to our resources, not sure how important it is

Reply:
actually that's an auto generated vector image, I don't think we add headers to the images